README.md

```diff
@@ -117,22 +120,10 @@ cd esp-idf
 $ git clone https://www.github.com/sieren/homepoint.git
 $ cd homepoint
 $ git submodule update --init --recursive
-$ mkdir build && cd build
-$ cmake ../
-$ make spiffs_spiffs_bin
-$ make -j 8 flash
```

Comment:
Let's keep the original version as it seems to work for everyonne with access to COM ports.